config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
+  resources :categories
+  resources :expenditures
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'home#index'
```

Comment:
- [ ] if I'm signed in I should see the category page as a home page not a logo of your app so please try to do that in order to make a better UX and also match the requirements 
- Screenshot of the requirements 
<img width="584" alt="Screen Shot 2022-10-14 at 18 26 04" src="https://user-images.githubusercontent.com/61790587/195905942-f36b9042-ef6a-4599-aff3-8c44724f562a.png">